File: Models/UserModel.cs
namespace SistemaElectoral.Models;

public class User
{
    public int IdUser { get; set; }
    public string Email { get; set; } = "";
    public string Password { get; set; } = "";
    public bool Active { get; set; }
    public PersonalData PersonalData { get; set; } = new();
    public Institute Institute { get; set; } = new();
    public Role Role { get; set; } = new();
}

File: server/src/models/user.model.js
class User {
    constructor({
        idUser, email, password, isActive, personalData, institute, role
    }) {
        this.idUser = idUser;
        this.email = email;
        this.password = password;
        this.isActive = !!isActive;
        this.personalData = personalData;
        this.institute = institute;
        this.role = role;
    }
}

module.exports = User;